frontend/integration-tests/tests/operator-hub/operator-hub.scenario.ts

```diff
@@ -24,166 +21,135 @@ describe('Subscribing to an Operator from OperatorHub', () => {
     await operatorHubView.operatorCommunityWarningIsClosed();
   });
 
-  it('shows the community operator when "Show Community Operators" is accepted', async() => {
-    await(catalogPageView.catalogTileById('etcd-openshift-marketplace').click());
+  it('shows the community Operator details when "Show Community Operators" is accepted', async() => {
+    const operatorTile = catalogPageView.catalogTiles.first();
+    await operatorTile.click();
     await operatorHubView.operatorCommunityWarningIsLoaded();
-    await operatorHubView.acceptCommunityWarningModal();
+    await operatorHubView.acceptForeverCommunityWarningModal();
     await operatorHubView.operatorCommunityWarningIsClosed();
 
     expect(operatorHubView.operatorModal.isDisplayed()).toBe(true);
-    expect(operatorHubView.operatorModalTitle.getText()).toEqual('etcd');
+    expect(operatorHubView.operatorModalTitle.getText()).toEqual(operatorTile.$('.catalog-tile-pf-title').getText());
+  });
 
+  it('filters Operators by catagory', async() => {
     await operatorHubView.closeOperatorModal();
     await operatorHubView.operatorModalIsClosed();
-  });
+    await catalogView.categoryTabs.get(1).click();
 
-  it('filters OperatorHub tiles by Category', async() => {
-    expect(catalogPageView.catalogTiles.isPresent()).toBe(true);
-    expect(catalogView.categoryTabs.isPresent()).toBe(true);
+    expect(catalogPageView.catalogTiles.count()).toBeGreaterThan(0);
   });
 
   it('displays subscription creation form for selected Operator', async() => {
-    await catalogPageView.catalogTileById('etcd-openshift-marketplace').click();
-    await operatorHubView.operatorCommunityWarningIsLoaded();
-    await operatorHubView.acceptCommunityWarningModal();
-    await operatorHubView.operatorModalIsLoaded();
+    await catalogView.categoryTabs.get(0).click();
+    installedOperator = await catalogPageView.catalogTiles.first().$('.catalog-tile-pf-title').getText();
+    await catalogPageView.catalogTileFor(installedOperator).click();
+    await browser.wait(until.visibilityOf(operatorHubView.operatorModal));
     await operatorHubView.operatorModalInstallBtn.click();
+    await operatorHubView.createSubscriptionFormLoaded();
 
-    expect(browser.getCurrentUrl()).toContain('/operatorhub/subscribe?pkg=etcd&catalog=community-operators&catalogNamespace=openshift-marketplace&targetNamespace=');
-    expect(operatorHubView.createSubscriptionFormTitle.isDisplayed()).toBe(true);
+    expect(operatorHubView.createSubscriptionFormName.getText()).toEqual(installedOperator);
   });
 
   it('selects target namespace for Operator subscription', async() => {
     await browser.wait(until.visibilityOf(operatorHubView.createSubscriptionFormInstallMode));
-    await $('input[value="singlenamespace-alpha"]').click();
 
-    expect($('input[value="SingleNamespace"]').getAttribute('disabled')).toBe(null);
+    if (operatorHubView.singleNamespaceInstallMode.getAttribute('disabled') !== null) {
```

Comment:
It's too bad we can't filter for one install mode or the other. It would be easy to break the things inside this `if` without realizing if the code doesn't run. I don't have a great suggestion, though. The alternative is to pick a known operator by name, but that obviously has problems, too.